Client/src/components/Shared/HeroBillboard/HeroBillboard.jsx

```
@@ -0,0 +1,102 @@
+        <div className="mb-4 flex items-center">
+          <span className="text-sm font-bold tracking-wide text-blue-400">{content.category}</span>
+        </div>
+        <h1 className="text-4xl md:text-5xl font-bold mb-3 text-white">{content.title}</h1>
+        <p className="text-lg mb-6 text-gray-300">{content.description}</p>
+        <div className="flex gap-3">
+          <button 
+            onClick={handlePlayDemo}
+            className="relative group flex items-center bg-blue-600 text-white px-6 py-3 rounded-full font-semibold shadow-lg"
+            type="button"
+          >
+            <span className="absolute inset-0 w-full h-full bg-gradient-to-r from-blue-500 to-blue-700 rounded-full shadow-md"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)